File: python/n551_StudentPre.py
# https://leetcode-cn.com/problems/student-attendance-record-i/
class Solution:
    def checkRecord(self, s: str) -> bool:
        return (s.count('A') <= 1 and s.count("LLL") == 0)
        
        
if __name__ == "__main__":
    pass

File: cpp/n551_StudentPre.cpp
// https://leetcode-cn.com/problems/student-attendance-record-i/
#include <iostream>
#include <vector>
#include <algorithm>
#include <string>
using namespace std;
class Solution {
public:
    bool checkRecord(string s) {
        if ((s.find('A') == s.npos || s.find('A') == s.rfind('A')) && s.find("LLL") == s.npos)
            return true;
        else
            return false;
    }
};

int main(){
    return 0;
}